src/firefly/js/templates/lightcurve/LcConverterFactory.js

```diff
@@ -97,14 +97,28 @@ const converters = {
         yNames: ['mag', 'tsv_flux'],
         yErrNames: ['magErr', 'tsv_fluxErr'],
         webplotRequestCreator: makeLsstSdssPlotRequest
+    },
+    [UNKNOWN_MISSION]: {
+        converterId: UNKNOWN_MISSION,
+        defaultImageCount: 3,
+        defaultTimeCName: 'mjd',
+        defaultYCname: 'value',
+        defaultYErrCname: '',
```

Comment:
There should be no default for these CName fields, right?  It has to be given via meta data.